assets/queries/terraform/aws/redis_not_compliant/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].resource.aws_elasticache_cluster[name] 
+  min_version_string = "4.0.10"
+  eval_version_number(resource.engine_version) <= eval_version_number(min_version_string)
```

Comment:
eval_version_number(resource.engine_version) < eval_version_number(min_version_string)

if it is equal to min version It should be good